Jenkinsfile

```diff
@@ -43,6 +26,12 @@ def cmake_build(compiler, flags, env4make, extradebugflags, prefixpath){
     if (prefixpath != "/usr/local") {
         configargs = "-DCMAKE_PREFIX_PATH=${prefixpath}"
     }
+    
+    if(flags.contains('-DBUILD_DEV=Off'))
```

Comment:
BUILD_DEV is off by default, so the absence of this flag doesn't mean that BUILD_DEV is enabled.

Reply:
I would like to be explicit in this file.

Reply:
And yes, you are correct, but let's follow the convention that `BUILD_DEV` must always be specified in Jenkinsfile. Then the text will be easier to understand. And there will be no need to insert an additional rule in the "if".

Reply:
>  And there will be no need to insert an additional rule in the "if".

There is no additional rule needed. You can write it as `if(!flags.contains('-DBUILD_DEV=On'))` and it will work whether its explicit or not.

Reply:
Thanks, I'll do that on next Jenksinfile update.

Reply:
resolved via #814